apps/projects/templates/a4_candy_projects/project_detail.html

```diff
@@ -1,324 +1,296 @@
+                                                        <div class="phase-info__item__description">{{ phase.description }}</div>
+                                                    </div>
+                                                {% endif %}
+                                            {% endfor %}
                                         </div>
                                     </div>
-                                    {% endif %}
-                                {% endfor %}
+                                {% endwith %}
+                            {% endif %}
+                        {% endif %}
+                    </div>
+                    {% if not view.is_project_view and not event %}
+                        <div class="container">
+                            <div class="row">
+                                {% if project.is_private %}
+                                    <div class="col-md-10 col-lg-8 offset-md-1 offset-lg-2 u-font-size-xs my-4">
+                                        <i class="fas fa-lock pe-2" aria-hidden="true"></i>
+                                        {% translate "This project is not publicly visible. Only invited users can see content and actively participate." %}
+                                    </div>
+                                {% elif project.is_semipublic %}
+                                    <div class="col-md-10 col-lg-8 offset-md-1 offset-lg-2 u-font-size-xs my-4">
+                                        <i class="fas fa-eye pe-2" aria-hidden="true"></i>
+                                        {% translate "This project is publicly visible. Invited users can actively participate." %}
+                                    </div>
+                                {% endif %}
                             </div>
                         </div>
-                        {% endwith %}
-                    {% endif %}
                     {% endif %}
-                </div>
-                {% if not view.is_project_view and not event %}
-                <div class="container">
-                    <div class="row">
-                    {% if project.is_private %}
-                        <div class="col-md-10 col-lg-8 offset-md-1 offset-lg-2 u-font-size-xs my-4">
-                            <i class="fas fa-lock pe-2" aria-hidden="true"></i>{% translate 'This project is not publicly visible. Only invited users can see content and actively participate.' %}
-                        </div>
-                    {% elif project.is_semipublic %}
-                        <div class="col-md-10 col-lg-8 offset-md-1 offset-lg-2 u-font-size-xs my-4">
-                            <i class="fas fa-eye pe-2" aria-hidden="true"></i>{% translate 'This project is publicly visible. Invited users can actively participate.' %}
+                    <!-- these blocks are only filled when in phase view -->
+                    {% if project_action %}
```

Comment:
This won't work as far as I know, there is no good way to check if a `block` has content or not. It sadly just means that we will have to have the empty div below